cobalt/android/apk/app/src/main/java/dev/cobalt/coat/CobaltActivity.java

```diff
@@ -328,9 +328,6 @@ public WebContents getActiveWebContents() {
   // TODO(b/375442742): re-enable native code.
   // private static native void nativeLowMemoryEvent();
 
-  // TODO(cobalt): make WebContent accessible in CobaltActivity or StarboardBridge.
```

Comment:
Can we add a bug number?

Reply:
I'm removing this comment. I haven't seen a need for ContentView from Activity yet. And WebContents is already available here.